release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1744,6 +1746,10 @@ Kubernetes 1.25 removed the following deprecated APIs, so you must migrate manif
 [.small]
 1. For more information about pod security admission in {product-title}, see xref:../authentication/understanding-and-managing-pod-security-admission.adoc#understanding-and-managing-pod-security-admission[Understanding and managing pod security admission].
 
+[id="ocp-4-12-empty-file-support-removed]
+==== oc registry command for empty file support
+`--registry-config` or `--to option` in the `oc registry login` command will stop accepting empty files (non existing files will still be valid use case) and writing to `-`(stdout) is also removed.
+
```

Comment:
FYI @bergerhoffer, @kalexand-rh, @JoeAldinger

Reply:
For context: https://issues.redhat.com/browse/OSDOCS-4418